libretro/libretro.cpp

```diff
@@ -397,6 +400,16 @@ static void check_variables(CoreParameter &coreParam)
       }
    }
 
+   bool isFastForwarding = environ_cb(RETRO_ENVIRONMENT_GET_FASTFORWARDING, &isFastForwarding);
+   if (isFastForwarding)
+   {
+      coreParam.fastForward = true;
+   }
+   else
+   {
+      coreParam.fastForward = false;
+   }
```

Comment:
Just replace this with coreParam.fastForward = isFastForwarding;

Reply:
Done.